src/main/java/foxtrot/uniform/charlie/kilo/service/implementation/AuthenticationServiceImpl.java

```diff
@@ -13,6 +15,8 @@
 public class AuthenticationServiceImpl implements AuthenticationService {
     @Inject
     private UserDao userDao;
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
```

Comment:
let's inject `userService` and `shoppingCartService` here - not dao

Reply:
Okay... I tried not to depend on service in other service.